src/main/java/io/github/redouane59/twitter/ITwitterClientV2.java

```diff
@@ -594,6 +594,14 @@ public interface ITwitterClientV2 {
    */
   TwitterList getList(String listId);
 
+  /**
+   * Get a tweet list by list id calling https://api.twitter.com/2/tweets
```

Comment:
are you sure about it ?
Isn't it https://api.twitter.com/2/lists/:id/tweets instead ?

Reply:
My bad. Yes it is lists/tweets one. I will fix doc

Reply:
Ok now I can understand better what you wanted to implement and it makes sense to have another method name :)